### PR TITLE
Server 10.10 Release Notes

### PR DESCRIPTION
**Pull request by pmaier1**
draft mode, still some things missing

### CODE REVIEW

File: modules/ROOT/pages/server_release_notes.adoc [hunk 1]
```diff
@@ -36,6 +37,51 @@
...
+* If the session_keepalive config option (`session_keepalive` in `config.php`) is set to `false`, a "heartbeat" request will be sent based on activity in order to extend the session lifetime. If there's no activity, the session might expire, and the user will need to login again.
+* A new config option `session_forced_logout_timeout` has been introduced in `config.php`. For advanced security, this option can be used to expire user sessions a configurable amount of seconds after they close the ownCloud browser tab or the whole browser, respectively, forcing the the user to log in again when they access ownCloud the next time. This option is disabled by default.
+
+=== Notable changes
+
+* The feature _Resend invitation mail_ in user management does not apply to guest users anymore as those have a different invitation flow than regular users (user-based vs. admin-based). https://github.com/owncloud/core/pull/40032[#40032]
+* Group administrators will now only see the groups they are an administrator of in user management (previously they also so other groups but were unable to manage them). https://github.com/owncloud/core/pull/39752[#39752]
```
Comment (phil-davis) [inline modules/ROOT/pages/server_release_notes.adoc:62]:
```suggestion
* Group administrators will now only see the groups they are an administrator of in user management (previously they also saw other groups but were unable to manage them). https://github.com/owncloud/core/pull/39752[#39752]
```

File: modules/ROOT/pages/server_release_notes.adoc [hunk 2]
```diff
@@ -36,6 +37,51 @@
...
+* The 'External Storage' feature (`files_external`) can now be disabled. https://github.com/owncloud/core/pull/39856[#39856]
+* General performance and stability improvements have been made. https://github.com/owncloud/core/pull/39847[#39847] https://github.com/owncloud/core/pull/40031[#40031]
+* The File Drop view for Upload-only public links has been visually improved to provide a better user experience. https://github.com/owncloud/core/pull/39900[#39900]
+* ownCloud Web: Guest users will now be explicitly listed as "Guests" (instead of "User") https://github.com/owncloud/core/pull/40013[#40013]
+
+=== Updated app versions
+
+TBC
+
+=== For developers
```
Comment (phil-davis) [inline modules/ROOT/pages/server_release_notes.adoc:77]: Maybe also mention 
"Enhancement - Expose user type of share receiver in share api: [#40013](https://github.com/owncloud/core/pull/40013)"
?

That is another addition to data returned in an API response.

File: modules/ROOT/pages/server_release_notes.adoc [hunk 3]
```diff
@@ -36,6 +37,51 @@
...
+
+* The configured session lifetime (`session_lifetime` in `config.php`) will now be reset each time a page is loaded or when a "heartbeat" request is sent.
+* If the session keepalive config option (`session_keepalive` in `config.php`) is set to `true`, a periodic "heartbeat" request will be made automatically regardless of any activity going on. This will reset the session lifetime preventing its expiration.
+* If the session_keepalive config option (`session_keepalive` in `config.php`) is set to `false`, a "heartbeat" request will be sent based on activity in order to extend the session lifetime. If there's no activity, the session might expire, and the user will need to login again.
+* A new config option `session_forced_logout_timeout` has been introduced in `config.php`. For advanced security, this option can be used to expire user sessions a configurable amount of seconds after they close the ownCloud browser tab or the whole browser, respectively, forcing the the user to log in again when they access ownCloud the next time. This option is disabled by default.
```
Comment (jnweiger) [inline modules/ROOT/pages/server_release_notes.adoc:57]:
```suggestion
* A new config option `session_forced_logout_timeout` has been introduced in `config.php`. For advanced security, this option can be used to expire user sessions a configurable amount of seconds after they close the ownCloud browser tab or the whole browser, respectively, forcing the user to log in again, when they access ownCloud the next time. This option is disabled by default.
```

Comment (jnweiger) [reply modules/ROOT/pages/server_release_notes.adoc:57]: duplicate the the

**Review by jnweiger — CHANGES_REQUESTED** [review]
One duplicate word 'the the' and one contradiction found.

File: modules/ROOT/pages/server_release_notes.adoc [hunk 4]
```diff
@@ -36,6 +37,51 @@
...
+
+=== Notable changes
+
+* The feature _Resend invitation mail_ in user management does not apply to guest users anymore as those have a different invitation flow than regular users (user-based vs. admin-based). https://github.com/owncloud/core/pull/40032[#40032]
+* Group administrators will now only see the groups they are an administrator of in user management (previously they also so other groups but were unable to manage them). https://github.com/owncloud/core/pull/39752[#39752]
+* ownCloud Server 10.9 introduced a new feature to xref:server_release_notes.adoc#highly-improved-initial-sync-and-discovery-performance[improve initial sync and discovery performance] which has been enabled by default. As there have been performance issues in certain environments, 10.10 disables the feature by default. It is recommended to enable the feature based on evaluations with test systems. https://github.com/owncloud/core/pull/40016[#40016]
```
[the remark this answers is not in the document]
Comment (pmaier1) [reply modules/ROOT/pages/server_release_notes.adoc:63]: Not really. In 10.9 it's enabled by default. 10.10 changes that to disabled by default.

**Comment by pmaier1** [comment]
This is ready for final review/merge.

**Review by mmattel — APPROVED** [review]
After a discussion with @pmaier1, the content ist fine, any changes needed can be fixed afterwards - approving

**Pull request merged**